codecs/avif/Makefile

```diff
@@ -42,6 +59,7 @@ $(OUT_ENC_MT_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMake
 			-DCONFIG_AV1_DECODER=0 \
 			-DCONFIG_AV1_HIGHBITDEPTH=0 \
 		" \
+		ENVIRONMENT="worker" \
```

Comment:
Hmm, can you try to make this a variable dependency? I think that would still work and allow to avoid duplication between targets here and there.

Something like:

```make
$(OUT_ENC_JS) $(OUT_ENC_MT_JS): ENVIRONMENT=worker
$(OUT_NODE_ENC_JS): ENVIRONMENT=node

$(OUT_ENC_JS) $(OUT_ENC_MT_JS) $(OUT_NODE_ENC_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLists.txt
    ...
    ENVIRONMENT=$(ENVIRONMENT) \
    ...
```

Reply:
(I see you already did this in other makefiles, should be easy to do the same here)